src/debuginfo.cpp

```diff
@@ -491,6 +499,12 @@ void lookup_pointer(DIContext *context, char **name, size_t *line,
     }
     *line = info.getLine();
     jl_copy_str(filename, info.getFileName());
+
+    if (inlineinfo.getNumberOfFrames() > 1) {
+        topinfo = inlineinfo.getFrame(inlineinfo.getNumberOfFrames() - 1);
+        jl_copy_str(inlined_file, topinfo.getFileName());
+        *inlined_line = topinfo.getLine();
+    }
```

Comment:
that would be nice, but i thought it was `info.Filename.c_str()` on the new versions?